client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx

```diff
@@ -30,6 +30,14 @@ export const ChapterPage: NextPage = () => {
           <Heading as="h5" fontWeight="normal">
             {data.chapter.name}
           </Heading>
+          <Box>
+            <span>
+              Chapter Users :
+              <Link href={`${id}/users`} target="_blank">
+                Users
```

Comment:
```suggestion
              <Link href={`${id}/users`} target="_blank">
                Chapter Users
```
I don't think we want Users to appear twice.